qa/tasks/cephfs/test_snap_schedules.py

```diff
@@ -588,12 +588,14 @@ def remove_snapshots(self, dir_path, sdn):
             self.mount_a.run_shell(['rmdir', snapshot_path])
 
     def get_snap_dir_name(self):
-        from tasks.cephfs.fuse_mount import FuseMount
-        from tasks.cephfs.kernel_mount import KernelMount
+        # from tasks.cephfs.fuse_mount import FuseMount
+        # from tasks.cephfs.kernel_mount import KernelMount
 
-        if isinstance(self.mount_a, KernelMount):
+        # if isinstance(self.mount_a, KernelMount):
+        if type(self.mount_a).__name__ == "KernelMount":
```

Comment:
This change is fine, but what's wrong with the import?

Reply:
I've no clue about what's wrong with the import. Haven't invested time to resolve the import issue.

BTW, is it imperative that the imports happen at runtime rather than at module load time ? i.e. can't we move the import statements at the top of the file with other imports

Reply:
We can move those imports, but there is nothing incorrect AFAICT with importing modules during runtime, but I could be wrong!